### DISCUSSION TITLE
Reactive java driver

### DISCUSSION THREAD

**Discussion by erhajdami** [ArcadeData/arcadedb#309] [Q&A]
Hi 

Is one jdbc or java specific reactive driver compliant with project reactor exists for ArcadeDb ??

Thanks

**Comment by lvca** [comment]
There is no specific driver to handle reactive programming, just a simple HTTP API call that uses Web Sockets:

https://docs.arcadedb.com/#HTTP-API

Basically, you register at the events you're interested in, like when records of a particular type are created, updated or deleted and ArcadeDB sends a push message to the client immediately.

If there was a generic driver to deal with reactive programming, I'd be happy to check it out and see if we can support ArcadeDB.

**Reply by erhajdami** [reply]
thks for the reply

I will come back in discussion after a migration from orientdb to arcadeb

**Comment by centoria** [comment]
What about now 10 month after my questions ??
There is a way to use mongodb reactive driver with arcadedb ??

Thanks

**Reply by lvca** [reply]
@centoria, unfortunately, this is a feature nobody else showed interest in, so it's a low priority for us unless anybody wants to contribute or there is sponsorship.
